### PR TITLE
DC track status: starts with 1 and the following values for each …

### PR DESCRIPTION
**Pull request by zieglerv**
…region.  The word has to be a short.

for each region: 
* 0: superlayer 1 is OK and superlayer 2 is OK (OK means no missing layers, i.e. 6 layers in the track segment has one or 2 hits)
* 1: superlayer 1 is OK and superlayer 2 has missing layers;  
* 2: superlayer 1 is OK and superlayer 2 is missing; 
* 3: superlayer 1 has missing layers and superlayer 2 is OK; 
* 4: superlayer 1 has missing layers and superlayer 2 has missing layers; 
* 5: superlayer 1 has missing layers and superlayer 2 is missing;
* 6: superlayer 1 is missing and superlayer 2 is OK; 
* 7: superlayer 1 is missing and superlayer 2 has missing layers; 
* 8: superlayer 1 is missing and superlayer 2 is missing; 8 should not happen at this stage

### CODE REVIEW

**Comment by zieglerv** [comment]
The code has been modified to write  a bit mask instead

File: reconstruction/dc/src/main/java/org/jlab/rec/dc/track/Track.java [hunk 1]
```diff
@@ -107,6 +107,52 @@ public void set_Status(int _Status) {
         this._Status = _Status;
     }
     
+    private int _BitStatus=0;
```
Comment (raffaelladevita) [inline reconstruction/dc/src/main/java/org/jlab/rec/dc/track/Track.java]: if the old status and related methods are not used any more, they should be removed and BitStatus could be renamed to Status

File: reconstruction/dc/src/main/java/org/jlab/rec/dc/track/Track.java [hunk 2]
```diff
@@ -107,6 +107,52 @@ public void set_Status(int _Status) {
...
+
+    public void setBitStatus(int bitStatus) {
+        this._BitStatus = bitStatus;
+    }
+    
+    private void setBitStatus() { 
+
+        int s[] = new int[6];//[slyr]-->0: OK; 1:missing layers; 2 missing
+        int sm = this.get_MissingSuperlayer()-1;
+        if(sm>0) s[sm] = 2;
```
Comment (raffaelladevita) [inline reconstruction/dc/src/main/java/org/jlab/rec/dc/track/Track.java]: Shouldn't it be sm>=0?

File: reconstruction/dc/src/main/java/org/jlab/rec/dc/track/Track.java [hunk 3]
```diff
@@ -107,6 +107,52 @@ public void set_Status(int _Status) {
...
+        int sl;
```
Comment (raffaelladevita) [inline reconstruction/dc/src/main/java/org/jlab/rec/dc/track/Track.java]: why declaring the variable outside the loop?

**Review by tongtongcao — APPROVED** [review]
The status setup looks fine.

**Pull request merged**